config.json

```diff
@@ -6,218 +6,265 @@
-      "slug": "robot-name",
+      "uuid": "3f649490-dc7d-4a77-a2a0-2ae71ae834a9",
+      "slug": "difference-of-squares",
       "core": false,
-      "unlocked_by": null,
-      "difficulty": 5,
+      "unlocked_by": "space-age",
+      "difficulty": 3,
       "topics": [
-
+        "Control-flow (loops)",
+        "Algorithms",
+        "Mathematics",
+        "Integers"
       ]
     },
     {
-      "uuid": "7a4fefd2-6e71-42d6-82fd-a25d2ef9eae9",
-      "slug": "beer-song",
+      "uuid": "5e5c48fb-91bb-495a-97f6-ec8642739a0a",
+      "slug": "word-count",
       "core": false,
-      "unlocked_by": null,
-      "difficulty": 6,
+      "unlocked_by": "pangram",
+      "difficulty": 1,
       "topics": [
-
+        "Control-flow (loops)",
+        "Lists",
+        "Strings",
+        "Unicode",
+        "Regular expressions"
       ]
     },
     {
-      "uuid": "32e79fd7-f002-4bdc-b6bd-7a91d8c1af61",
-      "slug": "grade-school",
+      "uuid": "2ceec21e-fb9d-4145-8788-826c3941eb01",
+      "slug": "anagram",
       "core": false,
-      "unlocked_by": null,
-      "difficulty": 6,
+      "unlocked_by": "pangram",
+      "difficulty": 2,
       "topics": [
-
+        "Strings",
+        "Filtering"
       ]
     },
     {
-      "uuid": "ab027a47-d483-4011-aac4-564a6284e5b8",
-      "slug": "etl",
+      "uuid": "bbd4665c-be4b-4d70-bc92-e91ce7a1d55f",
+      "slug": "phone-number",
       "core": false,
-      "unlocked_by": null,
-      "difficulty": 6,
+      "unlocked_by": "pangram",
+      "difficulty": 5,
       "topics": [
-
+        "Parsing",
+        "Transforming"
       ]
     },
     {
-      "uuid": "269577e5-e782-4264-9ad9-9ad4b8bc0aab",
-      "slug": "wordy",
+      "uuid": "7a4fefd2-6e71-42d6-82fd-a25d2ef9eae9",
+      "slug": "beer-song",
       "core": false,
-      "unlocked_by": null,
-      "difficulty": 6,
+      "unlocked_by": "bob",
+      "difficulty": 5,
       "topics": [
-
+        "Control-flow (conditionals)",
+        "Control-flow (loops)",
+        "Strings"
       ]
     },
+
     {
       "uuid": "8e7b92f4-a508-4200-bd62-b36281dd9ed9",
       "slug": "say",
       "core": false,
-      "unlocked_by": null,
+      "unlocked_by": "bob",
       "difficulty": 6,
       "topics": [
-        "pattern matching"
-      ]
-    },
-    {
-      "uuid": "2ceec21e-fb9d-4145-8788-826c3941eb01",
-      "slug": "anagram",
-      "core": false,
-      "unlocked_by": null,
-      "difficulty": 7,
-      "topics": [
-
+        "Control-flow (conditionals)",
+        "Control-flow (loops)",
+        "Mathematics",
+        "Integers",
+        "Exception handling",
+        "Strings",
+        "Text formatting"
       ]
     },
     {
       "uuid": "246129c9-83b5-43e0-beb6-8a2cea7e4e17",
       "slug": "food-chain",
       "core": false,
-      "unlocked_by": null,
-      "difficulty": 7,
+      "unlocked_by": "bob",
+      "difficulty": 4,
       "topics": [
-
+        "Text formatting",
+        "Algorithms"
       ]
```

Comment:
For the best experience on v2 we need to have about 3 exercises that are unlocked by hello world, otherwise the person doing the exercises will have only "leap" that they can work on after doing hello world, and at that point they will be stuck until someone gives them feedback on leap _and_ they iterate on it _and_ it gets approved _and_ they click "complete".

Reply:
I'm thinking that ideally we would have fewer core exercises and make sure that each core exercise has 2-4 bonus exercises that get unlocked by it, but I haven't actually raised this with the others yet. The discussion in v2-feedback after the beta launch of the prototype would suggest it, though.